src/vies-dotnet-api/Validators/FRVatValidator.cs

```diff
@@ -38,7 +38,11 @@ public FrVatValidator()
     protected override VatValidationResult OnValidate(string vat)
     {
         var validationKey = vat.Slice(0, 2);
-        var val = int.Parse(vat.Slice(2), CultureInfo.InvariantCulture);
+
+        if (!int.TryParse(vat.Slice(2), NumberStyles.Integer, CultureInfo.InvariantCulture, out var val))
```

Comment:
what is the maximum length for `FR` VAT code?

Reply:
Actually not sure. According to [taxid.pro](https://taxid.pro/docs/countries/france) it is FR + 11 digits.

Reply:
You want to check the length here?
This PR minimalistically avoids the exceptions on parsing.

Reply:
Sorry for my late response. 
I think it is better to check the length and throw VatValidationResult.Failed("Invalid FR vat: too long") instead of throwing it when fail to parse the input.

Reply:
No problem. I took also some time to see your comment 🙈
What about this?